apps/consent-ui/src/components/layouts/SideImageLayout/SideImageLayout.module.scss

```
@@ -0,0 +1,165 @@
+.container {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	@media (min-width: theme('screens.md')) {
+		flex-direction: row;
+	}
+}
+
+.mobileTabletHeader {
+	width: 100%;
+	background-image: linear-gradient(
+		270deg,
+		theme('colors.other.400'),
+		theme('colors.secondary.500')
+	);
```

I added this gradient to the tailwind config because it's used for the landing page and dashboard too, I think you would be able to use it here too:
```suggestion
	background: theme('backgroundImage.gradient.imageOverlay');
```